pkgs/by-name/cl/clash-nyanpasu/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, clash-verge
+, fetchurl
+}:
+clash-verge.overrideAttrs (old: rec {
+  pname = "clash-nyanpasu";
+  version = "1.4.5";
+
+  src = fetchurl {
+    url = "https://github.com/keiko233/clash-nyanpasu/releases/download/v${version}/clash-nyanpasu_${version}_amd64.deb";
+    hash = "sha256-cxaq7Rndf0ytEaqc7CGQix5SOAdsTOoTj1Jlhjr5wEA=";
+  };
+
+  # use builtin clash binaries, as clash-meta in nixpkgs is outdated
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    mv usr/* $out
+
+    runHook postInstall
+  '';
+
+  postFixup = "";
+
+  meta = old.meta // (with lib; {
```

Comment:
The license should be `gpl3Only` since the author has specified that the license is GPL3.0. This also applies to clash-verge-rev. Correct me if I'm wrong.

Reply:
Maybe there wasn’t this requirement at the earliest? Or maybe someone wrote it wrong in the first place.

Reply:
What I understand is the repo containing the license file can be plus, but if the author specifically states the version in the document and code, it is only. Is this statement right?

Reply:
Okay.